connector_jira/models/account_analytic_line/importer.py

```diff
@@ -81,7 +81,9 @@ def author(self, record):
                   'You must create a user or link it manually if the '
                   'login/email differs.') % (jira_author_key, email)
             )
-        employee = self.env['hr.employee'].search(
+        employee = self.env['hr.employee'].with_context({
```

Comment:
use kw args otherwise you replace the full ctx `with_context(active_test=False)`

Reply:
Thanks, actually never realized the difference before